backend/src/main/java/com/redhat/cloud/notifications/routers/EndpointResource.java

```diff
@@ -452,51 +539,65 @@ private static void checkSystemEndpoint(EndpointType endpointType) {
         }
     }
 
-    private String setupOpenBridgeProcessor(Endpoint endpoint, CamelProperties properties, String processorName) {
 
-        Map<String, Object> out = new HashMap<>();
-        out.put("name", processorName);
+    /*
+     * Creates a template processor object from the provided
+     * endpoint.
+     */
+    private Processor createProcessor(Endpoint endpoint) {
```

Comment:
How will processors work with user-defined templates? If each user is allowed to chose his template, does that mean that we will create one processor for each user template?

Reply:
That may be the consequence.